test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/runner/TestRunnerFeature.java

```diff
@@ -1,46 +0,0 @@
-package org.kie.cloud.tests.common.runner;
-
-import java.io.IOException;
-import java.util.Properties;
-
-import org.junit.rules.TestWatcher;
-import org.junit.runner.Description;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class TestRunnerFeature extends TestWatcher {
```

Comment:
Probably not correctly rebase git branch. We added this in PR #570

Reply:
I will make more attention to these issues in next PRs